java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -822,36 +674,20 @@ public NewArrayTreeImpl completeElementValueArrayInitializer(
 
     NewArrayTreeImpl elementValues = partial.isPresent() ?
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "commaToken" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)